IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -111,7 +111,12 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
         {
             MSID_LOG_INFO(self.requestParameters, @"Found valid access token, looking for refresh token...");
             NSError *rtError = nil;
-            id<MSIDRefreshableToken> refreshableToken = [self appRefreshTokenWithError:&rtError];
+            
+            // Trying to find FRT first.
+            id<MSIDRefreshableToken> refreshableToken = [self familyRefreshTokenWithError:&rtError];
```

Comment:
nit: Log rtError contents (currently ignored if FRT not found and there's non nil error), and add a verbose log to which kind of token we're returning.

Reply:
added more logs